pkg/discovery/discovery.go

```diff
@@ -80,6 +81,17 @@ func Run(kubeClient kubernetes.Interface, cfg *config.Config) []error {
 		errlst = append(errlst, err)
 	}
 
+	// 8. Always write the done file
+	doneFile := cfg.ResultsDir + "/done"
```

Comment:
There are a couple of benefits I saw:

1. The Dockerfile doesn't have to change